charts/ontology/templates/service.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+{{ include "ontology.labels" . | indent 4 }}
+spec:
+  ports:
+    - name: http
+      port: 8080
+  selector:
+    app: {{ .Chart.Name }}-service
```

```suggestion
  selector:
    app: {{ .Chart.Name }}
```
This field must be used to select a label that is present on ontology pods. 
Becasue all ontology pods just have the label `app: ontology` this is currently an empty service that doesn't select any pods.